druid/src/widget/button.rs

```diff
@@ -14,19 +14,81 @@
 
 //! A button widget.
 
-use crate::widget::prelude::*;
 use crate::widget::{Click, ControllerHost, Label, LabelText};
-use crate::{theme, Affine, Data, Insets, LinearGradient, UnitPoint};
+use crate::{theme, Affine, Color, Data, Insets, LinearGradient, UnitPoint};
+use crate::{widget::prelude::*, KeyOrValue};
 
 // the minimum padding added to a button.
 // NOTE: these values are chosen to match the existing look of TextBox; these
 // should be reevaluated at some point.
 const LABEL_INSETS: Insets = Insets::uniform_xy(8., 2.);
```

Comment:
Are we at the right point to reevaluate this yet? I've had some trouble getting a button / textbox / label to all have the same style. Would be nice if someone could use the same input values for each one and get the same results.

Reply:
Insets in the `TextBox` have been moved to the env, so I'd consider this comment outdated, and would resolve the insets for `Button` from the env as well.